### PR TITLE
ci(actions): require referenced issue and PR labels

### LINKED ISSUE #610: PRs must reference Issues

### LINKED ISSUE #611: PRs must have labels corresponding to conventional commits

### PR DESCRIPTION
**Pull request by andrewazores**
Fixes #610
Fixes #611

### CODE REVIEW

File: .github/workflows/linked-issue.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      - edited
+      - synchronize
+
+jobs:
+  verify-linked-issue:
+    runs-on: ubuntu-latest
+    if: github.actor != 'dependabot[bot]' && github.actor != 'dependabot-preview[bot]'
+    name: Verify Pull Request references Issue
+    steps:
+      - uses: Codeinwp/verify-linked-issue-action@6da9af91d78847c7ea264083a64eab3271b4369f
```
Comment (ebaron) [inline .github/workflows/linked-issue.yml:17]: Any particular reason for using this fork?

**Review by ebaron — COMMENTED** [review]
Apparently this action fails if there is no space after the issue number:
https://github.com/hattan/verify-linked-issue-action#known-issues

That could become annoying.

File: .github/workflows/linked-issue.yml (hunk 1 above)
Comment (andrewazores) [reply .github/workflows/linked-issue.yml:17]: I found the fork through this PR on the original repo:

https://github.com/hattan/verify-linked-issue-action/pull/30

In particular, the fork shouldn't suffer the known issues of the original.

Comment (ebaron) [reply .github/workflows/linked-issue.yml:17]: Oh nice. That includes the regex issue with the required trailing space? Either way I'm fine with giving it a try.

**Review by ebaron — APPROVED** [review]

File: .github/workflows/linked-issue.yml (hunk 1 above)
Comment (andrewazores) [reply .github/workflows/linked-issue.yml:17]: Yea, this PR itself has the action running on it and it successfully detected the two issues I referenced, neither of which were written with a trailing space.

**Pull request merged**